.github/workflows/devcontainer.yml

```
@@ -0,0 +1,34 @@
+name: devcontainer
+
+on:
+  # trigger manually
+  workflow_dispatch:
+  # run automatically every Monday at 01:00 UTC
+  schedule:
+  - cron: "0 1 * * 1"
```

```suggestion
on:
  push:
    paths:
      - '.devcontainer/**'
```
How about publishing updated images only when any one of the devcontainer files change?

---

Good idea. We can try it, but we still need the cronjob like thing to update when the base rust image changes

---

I've added the suggestion to also build on changes to the devcontainer folder. Commit b0dd53ccbb368eac19a33ba8c1365144c3c5b7c1